crdt/sequence.py

```diff
@@ -0,0 +1,247 @@
+from functools import cmp_to_key
+from re import S
+import uuid
+from enum import Enum
+
+from crdt.gset import GSet
+from crdt.gcounter import GCounter
+
+class Sequence():
+    """
+    Sequence is a CRDT that represents an ordered set of objects and supports insertion
+    and removal at arbitrary positions in the set.
+    """
+
+    def __init__(self, id=uuid.uuid4()):
+        self.id = id
+        self.operations = GSet()
+        self.clock = GCounter(self.id)
+        self.sequence = []
+
+    def compare_operations(self, a, b):
+        """
+        Compares this Operation to another Operation to determine ordering.
+        """
```

Comment:
Would you mind adding to the docstring what types `a` and `b` are here, as well as the expected return type?